tools/base/runner.py

```diff
@@ -86,3 +90,46 @@ def fork(self, *args, capture_output: bool = True, **kwargs) -> subprocess.Compl
         """Fork a subprocess, using self.context.path as the cwd by default"""
         kwargs["cwd"] = kwargs.get("cwd", self.context.path)
         return subprocess.run(*args, capture_output=capture_output, **kwargs)
+
+
+class BazelAdapter(object):
+
+    def __init__(self, context: "ForkingRunner"):
+        self.context = context
+
+    def query(self, query: str) -> list:
+        """Run a bazel query and return stdout as list of lines"""
+        resp = self.context.fork(["bazel", "query", f"'{query}'"])
```

Comment:
fork was already added - i named it that ~kinda arbitrarily - it encapsulates some defaults for suprocess - in particular running things from the provided (or otherwise) `path`

Reply:
Sure; can we rename to more standard Python subprocess terminology? I think`run` or `check_call` etc. are familiar.

Reply:
`run` is already used by this class and wouldnt tell you that its running *in a separate process* - check call does something slightly different and i think would be more confusing/unexpected to take the actual name of a subprocess function anyway

how about i rename it to just `subproc` or perhaps `subproc_run`

Reply:
Sure, `subproc_run` is good